docs/docs/step-06.md

```diff
@@ -152,6 +153,43 @@ Here, we use a recursive document splitter with a segment size of 100 tokens and
 
 Finally, we trigger the ingestion process and log a message when it's done.
 
+### In-memory embedding store
+
+If you are not able to run Dev Services with Docker or Podman, feel free to use the in-memory
+embedding store provided by LangChain4j.
+
+Add the following dependency to your `pom.xml` file:
```

Comment:
I don't think this is necessary, is it? It works for me without this.
Rather, I would tell the user to remove the `quarkus-langchain4j-pgvector` instead